site/styles.css

```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
 
 .button-container {
     display: flex;
+    justify-content: center;
 }
 
 .button-container .button {
+    display: inline-block;
     margin-right: 16px;
 }
 
 .button-container .button:last-child {
     margin-right: 0;
 }
 
+.back-to-top {
+    opacity: 1;
+    pointer-events: all;
+    position: fixed;
+    bottom: 6rem;
+    right: 3rem;
+    z-index: 9999;
+    width: 30px;
+    height: 30px;
+    text-align: center;
+    line-height: 30px;
+    background: #f1d608;
+    cursor: pointer;
+    border-radius: 2px;
+    transform: translate3d(0,0,0);
+    transition: all .3s ease;
+}
+
+.back-to-top:hover {
+    background: #f7de1b;
+}
+
+.back-to-top.hidden {
+    opacity: 0;
+    pointer-events: none;
+    transform: translate3d(300%, 0, 0);
+    transition: all .3s ease;
+}
+
 @media (min-width: 750px) {
+    body {
+        font-size: 1.8rem;
+    }
+    .page-header {
+        padding-top: 0;
+    }
+    .page-header__logo-link {
+        display: inline-block;
+        margin: 0 1.6rem 0 0;
+        width: 150px;
+        vertical-align: bottom;
+    }
+    .page-header nav {
+        display: inline-block;
+    }
+    .page-main {
+        width: 75%;
+    }
+    .page-header,
+    .card {
+        box-shadow: 0 14px 20px rgba(0,0,0,.3);
```

Comment:
This doesn't really seem to impact anyyhing due to the dark background, do we still need this?

Reply:
No, you're right, it's left over from before I changed the background... will remove.